bin/doctrine-migrations-migrate.sh

```diff
@@ -1,3 +1,3 @@
 #!/usr/bin/env bash
 
-php72 ./bin/console doctrine:migrations:migrate --env=prod --em=deploy
\ No newline at end of file
+php ./bin/console doctrine:migrations:migrate --env=prod --em=deploy
```

Comment:
If you removed the `php72` from the bin/console call in `doctrine-migrations-diff.sh` then you should be able to remove it here right?